client/packages/system/src/Stock/api/hooks/line/useRepacksByStockLine.ts

```diff
@@ -0,0 +1,12 @@
+import { useQuery } from 'packages/common/src';
+import { useStockApi } from '../utils/useStockApi';
+
+export const useRepacksByStockLine = (stockLineId: string) => {
+  const api = useStockApi();
+
+  const result = useQuery(api.keys.listRepackByStockLine(stockLineId), () =>
+    api.get.repacksByStockLine(stockLineId)
+  );
+
+  return { ...result };
```

Comment:
Is this possible instead?
```suggestion
  return useQuery(api.keys.listRepackByStockLine(stockLineId), () =>
    api.get.repacksByStockLine(stockLineId)
  );
```